versions.tf

```diff
@@ -1,3 +1,8 @@
 terraform {
-  required_version = ">= 0.12"
+  required_version = ">= 0.13"
+  required_providers {
+    mcaf = {
+      source = "terraform.schubergphilis.com/schubergphilis/mcaf"
+    }
+  }
```

Comment:
Did we already push this provider?

Reply:
Nope -- and this may need some discussion for consistency... 

Following the [In-House Providerd docs](https://www.terraform.io/docs/configuration/provider-requirements.html#in-house-providers) you specify your own source and can either host it or use that path in your local `terraform.d`. I decided `terraform.schubergphilis.com` was a reasonable value to use, so have this in the inception module:

```
terraform.d
└── terraform.d/plugins
    └── terraform.d/plugins/terraform.schubergphilis.com
        └── terraform.d/plugins/terraform.schubergphilis.com/schubergphilis
            └── terraform.d/plugins/terraform.schubergphilis.com/schubergphilis/mcaf
                └── terraform.d/plugins/terraform.schubergphilis.com/schubergphilis/mcaf/0.1.0
                    ├── terraform.d/plugins/terraform.schubergphilis.com/schubergphilis/mcaf/0.1.0/darwin_amd64
                    │   └── terraform.d/plugins/terraform.schubergphilis.com/schubergphilis/mcaf/0.1.0/darwin_amd64/terraform-provider-mcaf
                    └── terraform.d/plugins/terraform.schubergphilis.com/schubergphilis/mcaf/0.1.0/linux_amd64
                        └── terraform.d/plugins/terraform.schubergphilis.com/schubergphilis/mcaf/0.1.0/linux_amd64/terraform-provider-mcaf
```

And as mentioned in the other PR, need to specify the same provider source in each module.

Reply:
Guess we just need to publish the provider instead 😏

Reply:
It would make life easier 🙂 I thought I could publish it to the private registry in TFE but alas that's still for modules only...